src/lexers/julia.jl

```diff
@@ -65,10 +65,20 @@ function julia_is_string_macro(ctx::Context, count::Integer = 1)
 end
 julia_is_triple_string_macro(ctx::Context) = julia_is_string_macro(ctx, 3)
 
-# Julia Script Lexer.
+# Helper function to abstract differences in REPL.REPLCompletions across Julia versions
+function complete_keyword(s)
+    if isdefined(REPL.REPLCompletions, :complete_keyword)
+        return REPL.REPLCompletions.complete_keyword(s)
+    elseif isdefined(REPL.REPLCompletions, :complete_keyword!)
+        return REPL.REPLCompletions.complete_keyword!(REPL.REPLCompletions.Completion[], s)
+    else
+        error("This must be updated for new changes in REPL.REPLCompletions!")
+    end
+end
 
+# Julia Script Lexer.
 @lexer JuliaLexer let
-    keywords = [kw.keyword for kw in REPL.REPLCompletions.complete_keyword("")]
+    keywords = [kw.keyword for kw in complete_keyword("")]
```

Comment:
I think this can be simplified to:
```suggestion
    keywords = [kw.keyword for kw in REPL.REPLCompletions.sorted_keywords]
```

Reply:
Ah, good to know!